Allow setting the scaling group balancing policy.

ebabani opened the pull request.

xiaozhu36 commented:
Thanks for your contribute. Can you add a test case for this PR and submit the result of running this test case?

ebabani commented:
Added a test, small change so I added it to an existing test case. 

Results with a failing and passing run:

```
~/g/s/g/a/t/alicloud (ebabani/ess-multi-az-policy) $ env TF_ACC=1 go test -run TestAccAlicloudEssScalingGroup_vpc -v
=== RUN   TestAccAlicloudEssScalingGroup_vpc
--- FAIL: TestAccAlicloudEssScalingGroup_vpc (78.45s)
	testing.go:513: Step 0 error: Check failed: Check 7/7 error: alicloud_ess_scaling_group.foo: Attribute 'multi_az_policy' expected "BALANCEZ", got "BALANCE"
FAIL
exit status 1
FAIL	github.com/alibaba/terraform-provider/alicloud	78.488s
~/g/s/g/a/t/alicloud (ebabani/ess-multi-az-policy) $ env TF_ACC=1 go test -run TestAccAlicloudEssScalingGroup_vpc -v
=== RUN   TestAccAlicloudEssScalingGroup_vpc
--- PASS: TestAccAlicloudEssScalingGroup_vpc (213.03s)
PASS
ok  	github.com/alibaba/terraform-provider/alicloud	213.096s
~/g/s/g/a/t/alicloud (ebabani/ess-multi-az-policy) $
```

xiaozhu36 commented on alicloud/resource_alicloud_ess_scalinggroup.go:
```diff
@@ -77,6 +77,10 @@ func resourceAlicloudEssScalingGroup() *schema.Resource {
+			"multi_az_policy": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```
I think this new field 'multi_az_policy' needs a ValidateFunc and Default

ebabani replied:
Agreed. I'll make the change over the weekend.

xiaozhu36 commented on alicloud/resource_alicloud_ess_scalinggroup.go:
```diff
@@ -77,6 +77,12 @@ func resourceAlicloudEssScalingGroup() *schema.Resource {
+				Optional:     true,
+				Default:      "PRIORITY",
+				ValidateFunc: validateAllowedStringValue([]string{"PRIORITY", "BALANCE"}),
+			},
```
HI @ebabani Thanks for your contribution. Can you define two const parameters in the file `alicloud/extension_ess.go` to support these two optional values.

In addition, if this parameter doesn't support update, please set 'ForceNew' to it.
